### DISCUSSION TITLE
version of aggregation file for fast data

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#121] [Q&A] [answered]
Hi Mia,

I noticed that when creating a new single view creator the file version of each configuration, and especially the aggregation file, are all set to the `"version": "1.0.0"` instead of the latest version available (should be the `"version": "1.3.0"`).

Also in the documentation it is not available a clear paragraph that explains the features available for each version and the list of all the existing versions.

Thanks!

**Comment by ThisIsDemetrio** [comment, marked as answer]
Hi, @elmalakomar! 

You noticed right, the version set is the `1.0.0` while you should use (at the very least) the `1.1.0` (or the latest version, which is the `1.3.0`. Expect this to be updated in a future release of the console.

In the documentation page about the [Single View Creator Low Code configuration](https://docs.mia-platform.eu/docs/fast_data/configuration/single_view_creator/low_code#aggregation) it uses the `1.1.0`, but it is actually difficult to notice and, I agree with you with that, maybe the documentation page can be improved to reflect better these changes. We'll work on that.

Just to let you know:
- The v`1.1.0` supports many feature described in documentation, of the like of the _identifierQueryMapping_, recursive mappings, operators, etc.
- From v`1.1.0` to v`1.2.0`, you can add the _onStates_ operator inside your dependency in the `dependencies` operator to match one of the states of a document stored (either _PUBLIC_, _DRAFT_, _TRASH_ or _DELETED_).
- From v`1.2.0` to v`1.3.0`, comparison operator definition (the ones used inside the `and` and `or`) supports value coming from a file with the `__fromFile__` syntax.

Hope this helped.
If you need more info, let's discuss it here.
Cheers,
|D|

**Comment by elmalakomar** [comment]
Hi @ThisIsDemetrio 
I have also another question. I tried to manually set the version to the `1.3.0` but the console gave me the following error:
```
Unknown aggregation version
```

Here a screenshot of the behavior:

![image](https://user-images.githubusercontent.com/18056946/204558569-e3f05912-87ef-49c4-9a78-d2b0e86ce6d9.png)

The single view creator is using the following docker image: single-view-creator-plugin:5.6.0

Tell me if you need any more information.

Thanks

**Comment by ThisIsDemetrio** [comment]
It really sounds like a bug to me, caused by the validation in the Single View Creator low code configuration module. We're gonna open an issue on our board and fix this soon.

While we fix this issue, my suggestion to you would be either to use the v`1.2.0` or manually update the aggregation config map (but I wouldn't go with this last option if you don't really need the last version).
